### ISSUE TITLE
Sections with fp-noscroll won't trigger next section/slide on scroll on mobile

### ISSUE THREAD

**Issue by alvarotrigo** [alvarotrigo/fullPage.js#4505]
I still need to test it on different environments so I'll update the issue with more details later on.

**Comment by alvarotrigo** [comment]
It can be reproduced here from a phone (Chrome, Safari...) with fullpage.js 4.0.15:
https://codepen.io/alvarotrigo/pen/NWBegME

HTML code:

```html
<div id="fullpage">
    <div class="section fp-noscroll">
      <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
                <h1>Section 1</h1>
    </div>
    <div class="section fp-noscroll">
        <div class="slide" data-anchor="slide1">
          <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
                <h1>Section 2</h1>
        </div>
        <div class="slide" data-anchor="slide2">
          <h1>Slide 2.2</h1>
        </div>
    </div>
    <div class="section"> 
        <h2>Section 3</h2>
    </div>
</div>
``` 

JS
```js
new fullpage('#fullpage', {
  sectionsColor: ['yellow', 'orange', '#C0C0C0', '#ADD8E6'],
  
  // Get your license at https://alvarotrigo.com/fullPage/pricing/
  licenseKey: 'YOUR LICENSE KEY HERE '
});
```

**Comment by alvarotrigo** [comment]
This has been fixed on the new version 4.0.16! 🥳
